pyomo/contrib/pyros/pyros.py

```diff
@@ -375,6 +373,27 @@ def solve(self, model, first_stage_variables, second_stage_variables,
             setattr(model_data.original_model, cname, src_vars)
             model_data.working_model = model_data.original_model.clone()
 
+            # convert active objective to a minimization objective,
+            # if necessary
+            active_obj = list(
+                obj
+                for obj in model_data.working_model.component_data_objects(
+                    Objective,
+                    active=True,
+                    descend_into=True,)
+            )[0]
+            active_obj = recast_to_min_obj(model_data.working_model,
+                                           active_obj)
+
+            # remove inactive objectives, then deactivate the
+            # only remaining objective, of the working model
+            active_obj.deactivate()
+            remove_model_objectives(model_data.working_model,
+                                    keep_objectives=[active_obj])
```

Comment:
Why remove model objectives?  Deactivating objectives should be sufficient (they should just be ignored).